maldoca/base/file.cc

```diff
@@ -204,13 +207,12 @@ absl::Status Match(absl::string_view pattern,
 }
 #endif  // MALDOCA_CHROME
 
-// TODO(someone): Make this work with general file systems.
-absl::Status GetContents(const std::string& path, std::string* contents) {
+absl::Status GetContents(const base::FilePath& path, std::string* contents) {
```

Comment:
ditto can you do a wrapper with const base::FilePath::StringType for backward compat.